requirements.txt

```diff
@@ -1,12 +1,12 @@
 # Django/Framework Packages
 django>1.8,<1.9
 django-model-utils==1.4.0
+-e git+https://github.com/edx/django-rest-framework.git@3c72cb5ee5baebc4328947371195eae2077197b0#egg=djangorestframework==3.2.3
```

Comment:
Why is this here? All of the non-test-related packages need to go away. The first line of this file should simply be: `-e .`